docs/enterprise/orchestration.md

```diff
@@ -204,18 +221,202 @@ configs:
+* Check the [additional examples on GitHub](https://github.com/treeverse/charts/tree/master/examples/lakefs/enterprise) we provide for each authentication method (oidc, adfs, ldap, rbac, IAM etc).
+* The Database configurations between fluffy and lakeFS should be the same since they connect to the same DB.
+
+## Quickstart with the Kubernetes Helm Chart
+
+This examples contains no dependencies and it's the quickest way to start with lakeFS enterprise via Helm on any K8S cluster.
+The following values will deploy fluffy and lakeFS without SSO, using local blockstore and a dev postgres container.
+
+1. Create a `values.yaml` file with the following content and make sure to replace `<fluffy-docker-registry-token>`, `<lakefs.acme.com>` and `<ingress-class-name>`.
+1. In the desired K8S namespace run `helm install lakefs lakefs/lakefs -f values.yaml`
+1. In your browser go to the Ingress host to access lakeFS UI.
+
+```yaml
+lakefsConfig: | 
+  logging:
+      level: "DEBUG"
+  blockstore:
+    type: local
+ingress:
+  enabled: true
+  ingressClassName: <ingress-class-name>
+  annotations: {}
+  hosts:
+    - host: <lakefs.acme.com>
+      paths: 
+       - /
+fluffy:
+  enabled: true
+  image:
+    privateRegistry:
+      enabled: true
+      secretToken: <fluffy-docker-registry-token>
+  fluffyConfig: |
+    logging:
+      level: "DEBUG"
+  secrets:
+    create: true
+  sso:
+    enabled: false
+  rbac:
+    enabled: true
 
-**Note:** If you'd like to run this example on a k8s cluster, follow this section
-and replace fluffy and lakeFS configuration in the helm's `values.yaml` file.
+# useDevPostgres is true by default and will override any other db configuration, set false for configuring your own db
+useDevPostgres: true
+```
+
+## Deploy lakeFS Enterprise (Kubernetes)
+
+
+The following examples will guide you through the installation of lakeFS Enterprise using our [Helm Chart](#lakefs-helm-chart).
+
+### Prerequisites
+
+1. A KV Database, like postgres, should be configured and shared by fluffy and lakeFS.
+1. Access to configure SSO IdP, like Azure AD Enterprise Application.
+1. A proxy server should be configured to route traffic between the 2 servers (in K8S that is Ingress).
+1. Token for [dockerhub/fluffy](https://hub.docker.com/u/treeverse) Docker image. Please [contact us](mailto:support@treeverse.io) to get access to the Dockerhub image.
+    
+### Deploy lakeFS Enterprise with SSO
+
+
+Authentication in lakeFS Enterprise is handled by the Fluffy sso service which runs side-by-side with lakeFS. 
+The following examples are based on our [Helm Chart](#lakefs-helm-chart).
+
+For details on configuring the supported identity providers with Fluffy in-depth configuration see [SSO][lakefs-sso-enterprise-spec].
+
+**Note:** Full Fluffy configuration can be found [here]({% link enterprise/fluffy-configuration.md %}).
 {: .note }
 
-#### Azure Configuration
+* OpenID Connect
+* SAML
```

Comment:
No really, just the title too specific not mentioning that's SAML. 
Because  AD FS is like saying "Okta", The protocol is the important thing and it's SAML (typical choice for AD FS).